### ISSUE TITLE
"Invalid Savegame" on Cemu 2.1 with UKMM

### ISSUE THREAD

**Issue by Colorless-World** [marcrobledo/savegame-editors#460]
Hello. The save editor has worked fine for me untill I installed a mod called "The Trial of the Sword for Weenies". After I have installed it, the save editor always picks up all the saves as invalid. Removing the mod did not help. Here is a link to the google drive where I hosted the save files.

https://drive.google.com/file/d/1XT21wxOPAGUVbNe2UNTuqWJO8Enjz1Lh/view?usp=sharing

Thanks in advance.

**Comment by UhOhWhoops** [comment]
Was this ever fixed? I'm having a similar issue and I have a heavily modded BOTW on UKMM

**Comment by Colorless-World** [comment]
No, no luck. No one has ever spoken about this, and on the Modding Hub they
seemed to be as confused as me. I am still on the lookout for a fix of some
sort.

Il dom 10 mag 2026, 04:20 UhOhWhoops ***@***.***> ha scritto:
